mflowgen/full_chip/construct.py

```diff
@@ -296,6 +298,8 @@ def construct():
           g.connect_by_name( block, power_gdsmerge )
           g.connect_by_name( block, drc            )
           g.connect_by_name( block, lvs            )
+      # Tile_array can use rtl from rtl node
```

Comment:
Why weren't these connected already? Was that an oversight?

Reply:
nvm i guess this is because there are now inputs (potentially) being passed from the node